app/routes.py

```diff
@@ -1,2 +1,232 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import request, Blueprint, make_response
+from flask import jsonify
+from sqlalchemy import asc, desc
+import time
+import datetime
+import requests
+from flask import current_app as app
+import os 
+from app.models.goal import Goal
+
+path = "https://slack.com/api/chat.postMessage"
```

Comment:
*Suggestion*:  Make variables you want to stay stuck to a specific value, in all capital letters.  We normally make constants in all caps.  Also change the name to specify *which* path you want. 

```suggestion
SLACK_PATH = "https://slack.com/api/chat.postMessage"
```